src/App.jsx

```diff
@@ -80,6 +133,60 @@ class App extends Component {
       .map(([answer]) => answer);
 
     const { classes: { main } } = this.props;
+    if (locationOk !== LocationStatus.validLocation) {
+      clearTimeout(GPSLocationTimer);
+      if (!developerModeGPSCheck) {
+        return (
+          <MuiThemeProvider theme={theme}>
+            <CssBaseline />
+            <main className={main}>
+              <GPSCheck
+                developerModeGPSCheck={(developerMode) => {
+                  this.setState({ developerModeGPSCheck: { developerMode } });
+                }}
+                locationCheck={(onStation) => {
+                  this.setState({ locationOk: onStation });
+                  if (onStation === LocationStatus.validLocation) {
+                    GPSLocationTimer = setTimeout(
```

Comment:
**Suggestion:** There's something strange going on with these timers. I get location access prompts when I play the game in developer mode. Make sure these timers are properly stopped before starting new ones.

Reply:
This was intended